src/Api/Client.php

```diff
@@ -127,10 +124,43 @@ public function call($endPoint, $httpRequestMethod, $content = [])
 		return $response;
 	}
 
+    /**
+     * @param string
+     * @return resource|false
+     */
+    private function getCurlHandle($endPoint)
+    {
+        $url = self::URL.$endPoint;
+        if (array_key_exists($endPoint, $this->curlHandles)) {
```

Comment:
There is no point were the key will not exist because of your initial definition of the array `$curlHandles`.

Reply:
that is not right. because I cannot assume everyone using this function will use it with the correct endpoint.